journal/updated/01-journal.adoc

```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
-William Gibson did so judge him for his wickedness,
-that he kept him close in a room by himself.
-After five or six nights lying on the boards, I prevailed with him,
-that Friends might have the liberty of the town in the night,
-and be there in the morning.
+Then he asked me where I lived; I told him that when I was at home,
+I lived at Welch-pool, and my family was there.
+"`But,`" said he, "`where are you now?`"
+I asked him whether he knew that I also was a prisoner there with my friends;
+and he asked me whether I slept upon the boards with them; I told him I did.
+He said he was sorry for it, but went away in anger, being much discontented in himself.
+
+I did not see him again till next morning, at which time I went to him again,
+and discoursed kindly with him. He said he had enquired about me in town,
+and that I might have the liberty of the town if I wished.
+I acknowledged his kindness, but told him it would be no comfort to me
+to have the liberty of the town, and leave my friends and brethren there in prison.
+He said I might stay there in prison with them, then.
+So I did not see him till the next morning, and then I went to him again.
+He was in such a cross and bitter condition,
+that he would not allow any other Friend to speak to him.
+William Gibson condemned him for his wickedness,
+for which he kept him closely confined in a room by himself.
+After five or six nights of lying on the boards, I prevailed with him
+that Friends might have the liberty of the town for the night,
+and return to prison in the morning.
 So the next day he began to be more friendly to us.
 
 After some days,
 I desired our friend John Millington to come with me to the jailer`'s house,
-to see whether we could have leave to go home till the next court session;
-but it was not then granted; but he told me, if I pleased I might go home till then.
-I told him, he might as freely let them go as me, for most of them lived in the county,
-and I lived out of the county; but no more could we have that time of him.
-I was uneasy in myself, seeing I had got a place in him, to let him alone,
-and pressed for my friends further liberty.
-
-A little time afterwards, by serious arguments, as it was harvest time,
-and hard for their wives, or some of their families,
-to come with weekly necessaries for them, I, with my friend John Millington,
-prevailed with him to let them go,
-and he took our words for our appearance at the next court session.
-
-Through the goodness of God, we all returned together to Shrewsbury, to our prison,
+to see whether we could be granted the permission
+to go home till the next court session; but it was not then granted,
+although he told me that if I pleased I might go home till then.
+I told him he might as freely let them go as me, for most of them lived in the county,
+and I lived out of the county; but we could get nothing more from him at that time.
+I was uneasy in myself to leave him alone, seeing he had a regard for me,
```

Comment:
Let's reword this. Maybe: Seeing he had a regard for me, I felt uneasy in myself to let it drop,